### DISCUSSION TITLE
Can I open wallet list modal when connected? So that users can switch wallet

### DISCUSSION THREAD

**Discussion by zation** [family/connectkit#253] [Q&A]
The current modal will be profile modal when connected. And users must disconnect to see the wallet list modal. Is there anyway to simplify the switch wallet steps? For example: open wallet list modal when connected

**Comment by lochie** [comment]
Interesting suggestion. I see the UX suggestion could be useful, but just wondering if you there a bigger reason why a user wouldn't expect the disconnect + reconnect flow when wanting to change wallet?

Worth noting that with some wallets (e.g MetaMask, or wallets utilising WalletConnect 2.0) you can change which address you're connected with within the wallet UI.

**Reply by zation** [reply]
We just want to simplify the switch steps for our users. Disconnect + reconnect will be two steps.

For switching the address, the current experience will be ok. Our senario is switching the wallet, for example: switch from MetaMask to WalletConnet.
